scala-native-jdbc/src/main/scala/java/sql/ResultSet.scala

```diff
@@ -52,27 +58,13 @@ trait ResultSet extends AutoCloseable {
 
   // Column value getters by column name
   def getString(columnLabel: String): String
+  def getShort(columnLabel: String): Short
   def getInt(columnLabel: String): Int
   def getLong(columnLabel: String): Long
   def getDouble(columnLabel: String): Double
   def getFloat(columnLabel: String): Float
   def getBoolean(columnLabel: String): Boolean
   def getBytes(columnLabel: String): scala.Array[Byte]
-
-  /**
-   * Retrieves the number of columns in this ResultSet object.
-   */
-  def getColumnCount(): Int
-
-  /**
-   * Retrieves the name of the designated column.
-   */
-  def getColumnName(column: Int): String
-
-  /**
-   * Retrieves the designated column's SQL type.
-   */
-  def getColumnType(column: Int): Int
```

Comment:
I don't think these are part of [`java.sql.ResultSet`](https://docs.oracle.com/javase/8/docs/api/java/sql/ResultSet.html)